<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>problem 8.34 - Farrakhetdinova Azaliya</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../../../programming/test-/media/icon.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reynolds Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>124</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fanning Friction Factor area</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>91</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Re = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>155</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>f :   [ </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>154</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>154</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_0_edit">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>162</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_n_edit">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>162</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="re_edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>97</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>222</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>191</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Absolute Error</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="error_edit">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>228</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plot_button">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="solve_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_massage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>368</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solution</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fanning friction factor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>f = </string>
     </property>
    </widget>
    <widget class="QLabel" name="solution_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>unsolved</string>
     </property>
    </widget>
   </widget>
   <widget class="MplWidget" name="widget_graph" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>641</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
